.github/workflows/build_and_publish_ot_jar.yml

```diff
@@ -0,0 +1,28 @@
+name: OpenTelemetry Java Instrumentation JAR docker image builder
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  buildAndPublishOpenTelemetryJavaInstrumentationJarImg:
+    runs-on: ubuntu-latest
+    env:
+      AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+      AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.3.4
+
+      - name: Verify changes in opentelemetry-java-instrumentation-jar directory
+        uses: tj-actions/verify-changed-files@v8
+        id: verify-ot-jars
+        with:
+          files: |
+            opentelemetry-java-instrumentation-jar
```

Comment:
Are you sure that's the correct path? I don't see a file/directory like this in the repo